xmipy/xmi.py

```diff
@@ -22,35 +22,100 @@ class Xmi(Bmi):
 
     @abstractmethod
     def prepare_time_step(self, dt: float) -> None:
-        """ """
+        """Prepare a single time step.
+
+        Read data from input files and calculate the current time step length
+        and the simulation time at the end of the current time step.
+
+        Parameters
+        ----------
+        dt : float
+            MODFLOW time step length for the current time step. Currently
```

Comment:
There shouldn't be MODFLOW specific docs within xmipy

Reply:
modified to be generic